google-cloud-bigquery/lib/google/cloud/bigquery/data.rb

```diff
@@ -302,13 +302,13 @@ def all request_limit: nil
 
         ##
         # @private New Data from a response object.
-        def self.from_gapi gapi, table_gapi, service
-          formatted_rows = Convert.format_rows(gapi.rows,
+        def self.from_json gapi_json, table_gapi, service
```

Comment:
I would like to avoid names such as `to_json` and `from_json`, since they are commonly used in other libraries. What about `from_gapi_json`?

Reply:
Makes sense. Changed to `from_gapi_json`